### PR TITLE
express overhaul

### PR DESCRIPTION
**Pull request by BenAAndrew**
## Description
Changes made for this branch

## Checklist
- [x] All manual tests passing
- [x] All unit tests passing 
- [x] All UI tests passing
- [ ] Code reviewed by peer:\
&nbsp; - *No redundant whitespace*\
&nbsp; - *Indentation correct*\
&nbsp; - *Naming conventions followed*\
&nbsp; - *Comments added (where needed)*\
&nbsp; - *No commented out redundant code*\
&nbsp; - *Branch name correct*
- [ ] Code reviewed by TA

### CODE REVIEW

File: Express-API/express.js [hunk 1]
```diff
@@ -1,105 +1,82 @@
...
     response.send(reject)
   })
-});
+}
+
+app.post('/addrole', cors(), function (request, response) {
+	sendResponseData(rolesHandler.createRole(request.body, db), response)
+})
 
 app.post('/addcapability', cors(), function (request, response) {
```
Comment (jurassic-marc) [inline Express-API/express.js]: Double quote the url

File: Express-API/express.js [hunk 2]
```diff
@@ -1,105 +1,82 @@
...
 
 app.get("/bands", cors(), function(request, response) {
-  db.query(bandsQuery).then(rows => {
-    response.send(rows);
-  });
-});
+  sendResponseData(bandHandler.allBands(db), response)
+})
 
 app.post('/login', cors(), function (request, response) {
```
Comment (jurassic-marc) [inline Express-API/express.js]: Double quote the url

File: Express-API/express.js (hunk 1 above)
Comment (BenAAndrew) [reply Express-API/express.js]: sorted

File: Express-API/express.js [hunk 3]
```diff
@@ -1,105 +1,82 @@
...
+  sendResponseData(rolesHandler.getRolesInDept(request.query, db), response)
+})
 
 app.get("/capabilities", cors(), function(request, response) {
-  db.query(capabilitiesQuery,[request.query.departmentID]).then(rows => {
-    response.send(rows);
-  });
-});
+  sendResponseData(capabilityHandler.getCapabiltiies(request.query, db), response)
+})
```
Comment (jamescrone) [inline Express-API/express.js]: nice job refactoring this, looking a lot tidier

Comment (BenAAndrew) [reply Express-API/express.js]: cheers

**Pull request merged**